suite/suite.go

```diff
@@ -179,16 +179,20 @@ func Run(t *testing.T, suite TestingSuite) {
 }
 
 func runTests(t testing.TB, tests []testing.InternalTest) {
-	r, ok := t.(runner)
-	if !ok { // backwards compatibility with Go 1.6 and below
-		if !testing.RunTests(allTestsFilter, tests) {
-			t.Fail()
+	if len(tests) == 0 {
```

Comment:
Would it make sense to implement this as a guard clause instead? 

Something like:
```
if len(tests) == 0 {
    t.Log(...)
    return
}
```

That way we don't have to indent the rest of the function in an `else` block

Reply:
Works for me. Done.